etc/auto/config

```diff
@@ -14,6 +14,17 @@ if [ "$HWE_X11" = "yes" ]; then
     XORG_HWE="xserver-xorg-hwe-${BASEVERSION}"
 fi
 
+case "$ARCH" in
+    amd64|i386)
+        MIRROR_BINARY_URL="http://archive.ubuntu.com/ubuntu/"
+        MIRROR_BINARY_SECURITY_URL="http://security.ubuntu.com/ubuntu/"
+        ;;
+    arm64)
+        MIRROR_BINARY_URL="http://ports.ubuntu.com/ubuntu-ports/"
+        MIRROR_BINARY_SECURITY_URL="http://ports.ubuntu.com/ubuntu-ports/"
+        ;;
+esac
+
```

Comment:
Maybe we may want to put these variables into terraform*.conf and remove the case sentence here

Reply:
I understand your suggestion; however, `MIRROR_URL` currently varies across `etc/terraform*.conf` files (official or Azure source), while these `MIRROR_BINARY_URL` and `MIRROR_BINARY_SECURITY_URL` should use the official source and remain consistent. Moving them into `etc/terraform*.conf` files would introduce duplication, conflicting with the *single source of truth* principle.